src/Page.js

```diff
@@ -409,7 +411,7 @@ class Page {
  * @param pageData a page with its front matter collected
  */
   generateExpressiveLayout(pageData, fileConfig) {
-    const nj = nunjucks;
+    const nj = _nunjucks;
```

Comment:
Nunjucks instance in expressive layouts should require date filter as well I believe.